ImagineEngine.xcodeproj/project.pbxproj

```diff
@@ -1703,6 +1721,7 @@
 				PRODUCT_NAME = ImagineEngine;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
```

Comment:
This shouldn't be here 😄

Reply:
@JohnSundell I added the `DEBUG` value for `SWIFT_ACTIVE_COMPILATION_CONDITIONS ` for the `DEBUG`build configurations in all the targets to make sure that I can do `#if DEBUG` in the code to execute enclosing code only in debug modes in the targets. Did I get this requirement incorrectly. Or is there a better approach to this?

Reply:
@JohnSundell I added the `DEBUG` value for `SWIFT_ACTIVE_COMPILATION_CONDITIONS` for the `DEBUG` build configurations in all the targets to make sure that I can do `#if DEBUG` in the code to execute enclosing code only in debug modes in the targets. Did I get this requirement incorrectly. Or is there a better approach to this?

Reply:
Ah my bad, thought it was something Xcode auto-added 😅